starport/services/chain/plugin-launchpad.go

```diff
@@ -202,3 +221,7 @@ func (p *launchpadPlugin) StoragePaths() []string {
 func (p *launchpadPlugin) GenesisPath() string {
 	return fmt.Sprintf(".%s/config/genesis.json", p.app.nd())
 }
+
+func (p *launchpadPlugin) Version() cosmosver.MajorVersion { return cosmosver.Launchpad }
+
+func (p *launchpadPlugin) SupportsIBC() bool { return true }
```

Comment:
```suggestion
func (p *launchpadPlugin) SupportsIBC() bool { return false }
```

It should return `false` if IBC is not supported ^^

Reply:
:sweat_smile: right!..